### PR TITLE
Port the detail controller .refresh() function to the list controller

### PR DESCRIPTION
**Pull request by davidmiller**

### CODE REVIEW

File: opal/static/js/test/patient_list.controller.test.js [hunk 1]
```diff
@@ -207,8 +207,30 @@ describe('PatientListCtrl', function() {
 
     });
 
+  describe('refresh()', function() {
+      fit('should update .episodes with what the server thinks is the episode', function() {
```
Comment (fredkingham) [inline opal/static/js/test/patient_list.controller.test.js]: witness the fitness?

(just it)

Comment (davidmiller) [reply opal/static/js/test/patient_list.controller.test.js]: Good shout - and you got there first with the fix too :)

**Comment by pacharanero** [comment]
are we happy to merge so I can test downstream epidur.io foo?

**Comment by pacharanero** [comment]
tested locally with the use case in Epidur.io, all working

**Pull request merged**